src/components/Configure/nav/ObjectManagementNav/v2/Tabs/index.tsx

```diff
@@ -111,8 +111,7 @@ export function VerticalTabs({
           <OtherTab
             completed={otherNavObject.completed}
             pending={objectConfigurationsState?.other?.write?.isWriteModified || false}
-             // if read tab exists, display 'other' else 'write' when write tab is the only tab
-            displayName={readNavObjects.length ? 'Other' : 'Write'}
+            displayName="Write"
```

Comment:
This is the only substantive change, everything else is just renaming variables.